README.md

```diff
@@ -247,6 +247,12 @@ the following command:
 $ docker run -p 4140:4140 -p 9990:9990 -v /absolute/path/to/myapp:/myapp buoyantio/linkerd:0.0.10-SNAPSHOT /myapp/linkerd.yml
 ```
 
+We supply a config that routes to a single backend on localhost:8080 for your convenience.
+
+```
+$ docker run -p 4140:4140 -p 9990:9990 -v /path/to/linkerd/linkerd/examples:/config buoyantio/linkerd:0.7.1-SNAPSHOT /config/static_namer.yaml
```

Comment:
Ok, I have changed the wording to make it clear this is for local testing convenience. I added this config because I wanted to test running the container with a simple static config and I felt new users would appreciate that.

Happy to make other changes.